plugins/service_discovery/srv.md

```diff
@@ -0,0 +1,105 @@
+# SRV Service Discovery Plugin
+
+The `srv` serivce discovery plugin updates targets by [SRV Record](https://tools.ietf.org/html/rfc2782)
+
+## Example Configuration
+
+This is an exmple with out\_forward.
+It udpates targets to get SRV record from `_fluentd._tcp.exmaple.com`.
+
+```
+<source>
+  @type forward
+
+  <service_discovery>
+    @type srv
+    service fluentd
+    proto tcp
+    hostname example.com
+  </service_discovery>
+</source>
+```
+
+## Parameters
+
+### @type
+
+The value must be `srv`.
+
+### service
+
+| type   | default                 | version |
+|:-------|:------------------------|:--------|
+| string | required parameters     | 1.10.0  |
+
+Service without underscore in [RFC2782](https://tools.ietf.org/html/rfc2782)
+
+### proto
+
+| type   | default                 | version |
+|:-------|:------------------------|:--------|
+| string | required parameters     | 1.10.0  |
+
+Proto without underscore in [RFC2782](https://tools.ietf.org/html/rfc2782)
+
+### hostname
+
+| type   | default                 | version |
+|:-------|:------------------------|:--------|
+| string | required parameters     | 1.10.0  |
+
+Name in [RFC2782](https://tools.ietf.org/html/rfc2782)
+
+
+### dns\_server\_host
+
+| type   | default | version |
+|:-------|:------------------------|:--------|
+| string | nil     | 1.10.0  |
+
+Hostname of DNS server to request the SRV record
+
+### interval
+
+| type   | default | version |
+|:-------|:--------|:--------|
+| integer | 60     | 1.10.0  |
+
+Interval of requesting to DNS server
+
+### dns\_lookup
+
+| type   | default | version |
+|:-------|:------------------------|:--------|
+| integer | 60     | 1.10.0  |
```

Comment:
This is bool.

Reply:
thanks 🙏 https://github.com/fluent/fluentd-docs-gitbook/pull/160/commits/fd5bea540af51fafb51710f5af239b58bef5f977#diff-0e07f0f935fc7c0757b985c04deedd13R74